### PR TITLE
Bump version to 0.6.1

### PR DESCRIPTION
**Pull request by AlexWaygood**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -1,5 +1,23 @@
...
 
+## 0.6.1
+
+### Preview features
+
+- \[`fastapi`\] Implement `fast-api-unused-path-parameter` (`FAST003`) ([#12638](https://github.com/astral-sh/ruff/pull/12638))
+
+### Rule changes
+
+- \[`pylint`\] Rename too-many-positional(-arguments) (`R0917`) ([#12905](https://github.com/astral-sh/ruff/pull/12905))
```
Comment (dhruvmanila) [inline CHANGELOG.md:11]:
```suggestion
- \[`pylint`\] Rename `too-many-positional` to `too-many-positional-arguments` (`R0917`) ([#12905](https://github.com/astral-sh/ruff/pull/12905))
```

File: CHANGELOG.md [hunk 2]
```diff
@@ -1,5 +1,23 @@
 # Changelog
 
+## 0.6.1
+
```
Comment (MichaReiser) [inline CHANGELOG.md:4]: Should we mention the pre-commit change here, considering that pre-commit package does not have its own changelog

**Review by dhruvmanila — APPROVED** [review]

**Review by MichaReiser — APPROVED** [review]

File: CHANGELOG.md (hunk 2 above)
Comment (AlexWaygood) [reply CHANGELOG.md:4]: I considered this. I'll add a note

Comment (dhruvmanila) [reply CHANGELOG.md:4]: We could update the release notes once it's "auto" released?

Comment (AlexWaygood) [reply CHANGELOG.md:4]: I added a note about the pre-commit issue; how's that look?

**Pull request merged**